Makefile

```diff
@@ -21,6 +21,13 @@ README.md: index.js
 lint:
 	$(JSHINT) -- index.js test/index.js
 	$(JSCS) -- index.js test/index.js
+	@echo 'Checking for missing link definitions...'
+	grep -o '\[[^]]*\]\[[^]]*\]' index.js \
+	| sort -u \
+	| sed -e 's:\[\(.*\)\]\[\]:\1:' \
+	      -e 's:\[.*\]\[\(.*\)\]:\1:' \
+	      -e '/0-9/d' \
+	| xargs -I '{}' sh -c "grep '^//[.] \[{}\]: ' index.js"
```

Comment:
I noticed that __index.js__ did not include a link definition for "Foldable". This should catch such omissions in future.

This may be the first non-trivial Unix pipeline I have composed on OS X which has worked first time on Linux!